Dockerfile_multi_folder

```diff
@@ -0,0 +1,14 @@
+ARG BASE_IMAGE=nansencenter/geospaas:2.0.0-slim
```

Comment:
The name of this file give no idea of what it's for.
If we go with the CLI thing, this dockerfile could be named Dockerfile_cli, and the other one Dockerfile_worker.

Reply:
Done in the next commit.